libr/util/print.c

```diff
@@ -2570,6 +2570,37 @@ R_API void r_print_hex_from_bin(RPrint *p, char *bin_str) {
 	free (buf);
 }
 
+R_API void r_print_bin_from_str(RPrint *p, char *str) {
```

Comment:
i think its better to make this function return a string instead of void. so we can't move it away from the `RPrint` namespace.. and find a better name for it like `RMem.to_binstring()`. you can use RStrBuf internally to construct the string and then just `return r_strbuf_drain()`